plugins/cloudconfigs/azure/plugin/azure.go

```diff
@@ -0,0 +1,94 @@
+package plugin
+
+import (
+	"fmt"
+
+	"github.com/enaml-ops/enaml"
+	"github.com/enaml-ops/omg-cli/plugins/cloudconfigs"
+	"github.com/enaml-ops/omg-cli/plugins/cloudconfigs/azure/plugin/generated"
+	"gopkg.in/urfave/cli.v2"
+	"gopkg.in/yaml.v2"
+)
+
+type NetworkCloudProperties struct {
+	NetworkName string `yaml:"virtual_network_name"`
+	SubnetName  string `yaml:"subnet_name"`
+}
+type AzureCloudConfig struct {
+	Context *cli.Context
+}
+
+//NewAzureCloudConfig : Create Azure cloud config
```

Comment:
This would be a bit more idiomatic:
`// NewAzureCloudConfig creates an Azure cloud config from the specified context`

Good resource for writing godoc: https://blog.golang.org/godoc-documenting-go-code